Travis

peabody124 opened the pull request:
Support for travis-ci build verification

peabody124 committed:
Travis: check in build configuration

peabody124 committed:
Travis: no need to manually install arm-sdk

tracernz commented:
FYI: I played with Travis last year and found our build regularly took too long and timed out, not sure if they have changed the time limit since then.

peabody124 committed:
Travis: test OSX and Linux

peabody124 committed:
Travis: use dual core for packaging

peabody124 committed:
Travis: for now do not package

Packaging assumes that we install into the Qt into the
default directories, which prohibits using system packages
(which is easier with travis). For now just use travis for
testing the build which just requires compiling things.

peabody124 committed:
Travis: include more libraries

peabody124 committed:
Travis: exclude OSX for now
